src/pages/gen2/build-a-backend/functions/extend-with-cdk/index.mdx

```diff
@@ -0,0 +1,17 @@
+export const meta = {
+  title: 'Extend with CDK',
+  description:
+    'Learn how to extend the underlying Function resources'
+};
+
+export function getStaticProps() {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+## Adding IAM Policies
+
+{/* ## Lambda Layers */}
```

Comment:
Shouldn't this be a top level page? Whether it's done via CDK or not is secondary.